en/sensor/bitcraze_flow.md

```diff
@@ -0,0 +1,67 @@
+# Bitcraze Optical Flow Board
+
+
+
+
+
+{% youtube %}
+https://youtu.be/0Jpq6DU_HVg
+{% endyoutube %}
+
+
+## Where to Buy
+
+You can order this module from the [Bitcraze website](https://www.bitcraze.io/flow-breakout/).
+
+
+## Specifications
+
+
+
+
+## Pixhawk Setup
+
+In order to connect the bitcraze flow board to the Pixhawk you will need to solder the wires of the Pixhawk SPI
+cable to the flow board. The SPI cable has 7 wires from which you need to connect 6 to the flow board. The diagram below shows you which wires you need to solder to which location on the flow board.
+
+| Pixhawk SPI port (from left to right) | Bitcraze flow board |
+| -- | -- |
+| 1 (VCC)  | VCC |
+| 2 (SCK)  | CLK |
+| 3 (MISO) | MISO |
+| 4 (MOSI) | MOSI |
+| 5 (CS1) | Do not connect |
+| 6 (CS2) | CS |
+| 7 (GND) | GND |
+
+
+> **Note** Applies to all PX4 FMU versions, including all versions of Pixhawk.
+
+- Update the firmware on PX4Flow using *QGroundControl* (in the top left menu, click on CONFIG, then on Firmware Upgrade)
+- Connect PX4Flow I2C to the Pixhawk I2C
+
+The module will be detected on boot.
+Flow data should be coming through at 10Hz if the autopilot is connected via **USB**. 
+Flow data is transmitted via wireless at a lower rate.
+
+
+### Mounting/Orientation
+
+The recommended mounting orientation (meaning: zero rotation) is defined as Y on flow board pointing towards **front of vehicle**, as shown in the following picture.
+
+![PX4Flow align with Pixhawk](../../assets/hardware/sensors/px4flow/px4flowalignwithpixhawk.jpg)
```

Comment:
Should change image of your actual hardware .. and globally search for PX4Flow  to see where still used (replace)

Reply:
Ideally image using more recent pixhawk device than old 3DR Pixhawk.